appstore/validator_test.go

```diff
@@ -318,3 +320,19 @@ func serverWithResponse(statusCode int, response string) http.Handler {
+	token := jwt.Token{}
+	if err := client.ParseNotificationV2(tokenStr, &token); err != nil {
+		t.Errorf("expected error to be not nil since tokenStr is legal")
```

Comment:
It seems the error sentence should be `Expected an error to be nil since the tokenStr is legal`